### PR TITLE
Add cookie with value of pl setting.

### PR DESCRIPTION
**Pull request by hibooboo2**
This is used by ui to display generic or not ui.

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
Is it possible to put this in a request handler?   It just seems dirty to put it here.  It's not core to the framework.

File: code/framework/api/src/main/java/io/cattle/platform/api/servlet/ApiRequestFilter.java [hunk 1]
```diff
@@ -177,4 +183,27 @@ public WrappedException(Throwable cause) {
...
+        if (httpRequest.getCookies() != null) {
+            for (Cookie c : httpRequest.getCookies()) {
+                if (PL.equals(c.getName()) && c.getName() != null) {
+                    plCookie = c;
+                    break;
+                }
+            }
+        }
+
+        if (plCookie == null) {
```
Comment (ibuildthecloud) [inline code/framework/api/src/main/java/io/cattle/platform/api/servlet/ApiRequestFilter.java]: Can this just be any or statement and not duplicate the blocks.

**Comment by ibuildthecloud** [comment]
@hibooboo2 Nevermind about the request handler comment, you probably need to keep the code here.

File: code/framework/api/src/main/java/io/cattle/platform/api/servlet/ApiRequestFilter.java (hunk 1 above)
Comment (hibooboo2) [inline code/framework/api/src/main/java/io/cattle/platform/api/servlet/ApiRequestFilter.java]: yes

**Comment by hibooboo2** [comment]
Tried to put it in handler needs to be here before the ui.

**Comment by vincent99** [comment]
Results LGTM, Java LJTM

**Pull request merged**